pkgs/development/libraries/xxHash/default.nix

```
@@ -11,21 +17,25 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-2WoYCO6QRHWrbGP2mK04/sLNTyQLOuL3urVktilAwMA=";
   };
 
-  # Upstream Makefile does not anticipate that user may not want to
-  # build .so library.
-  postPatch = lib.optionalString stdenv.hostPlatform.isStatic ''
-    sed -i 's/lib: libxxhash.a libxxhash/lib: libxxhash.a/' Makefile
-    sed -i '/LIBXXH) $(DESTDIR/ d' Makefile
-  '';
+  # Merged in https://github.com/Cyan4973/xxHash/pull/649
+  # Should be present in next release
+  patches = [(fetchurl {
```

Something like this might be cleaner in case another patch is added in the future:

```suggestion
  patches = [
    # Merged in https://github.com/Cyan4973/xxHash/pull/649
    # Should be present in next release
    (fetchurl {
```